WTWSMS/history/titles/c_adana.txt

```diff
@@ -13,6 +13,14 @@
 500.1.1={
 	holder=6860
 }
+524.1.1={
+    holder=1310010
+    liege="k_cilicia"
+}
+527.1.1={
+    holder=0
```

Comment:
Once again, you can't have holder = 0 on a county.

Reply:
Solved all of them.